plugins/initLaunchDarkly.ts

```diff
@@ -1,8 +1,23 @@
-import { initialize, LDClient, LDOptions } from 'launchdarkly-js-client-sdk'
+import { initialize, LDClient, LDFlagSet, LDOptions, LDUser } from 'launchdarkly-js-client-sdk'
 
 // get rid of "element implicitly has an 'any' type..."
 declare const window: any
 
+/**
+ * Default flag values when LD is not available.
+ * NB: Uses "business-edit" project (per LD client id in config).
+ */
+const defaultFlagSet: LDFlagSet = {
+  'banner-text': '', // by default, there is no banner text
+  'enable-digital-credentials': false,
+  'sentry-enable': false, // by default, no sentry logs
+  'show-alert-phone-numbers-firm': false,
+  'supported-business-summary-entities': [],
+  'supported-correction-entities': [],
+  'supported-dissolution-entities': [],
+  'supported-consent-continuation-out-entities': []
+}
```

Comment:
These are the default flags for Filings UI or something. Fill this in with whatever flags you are using. Also update comment on line 8.

I just looked at LaunchDarkly and I don't see a name-examination "project". (There is a namex project but I think it's used by the API.) Please work with @pwei1018 to create a suitable LD project and your initial feature flags.